### PR TITLE
feat: sign data with a magic prefix for security

### PR DESCRIPTION
**Pull request by zhangyouxin**
# What Changed

- `wallet_fullOwnership_signData()` should be sign with [magic prefix bytes]

## Change Type

Indicate the type of change your pull request is:

- [ ] `documentation`
- [ ] `patch`
- [x] `minor`
- [ ] `major`

### CODE REVIEW

**Comment by homura** [comment]
@zhangyouxin can you help to resolve conflicts

File: packages/extension-chrome/__tests__/services/ownership/fullOwnership.test.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import { bytes } from '@ckb-lumos/codec';
 import { Script, utils, Transaction } from '@ckb-lumos/lumos';
 import { common } from '@ckb-lumos/common-scripts';
 import { createEventHub } from '../../../src/services/event';
+import { SIGN_DATA_MAGIC } from '@nexus-wallet/protocol/lib/constants';
```
Comment (homura) [inline packages/extension-chrome/__tests__/services/ownership/fullOwnership.test.ts:23]:
```suggestion
import { SIGN_DATA_MAGIC } from '@nexus-wallet/protocol';
```

File: packages/extension-chrome/src/services/ownership/fullOwnership.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { BackendProvider } from './backend';
 import { HexString, Script, utils } from '@ckb-lumos/lumos';
 import { common } from '@ckb-lumos/common-scripts';
 import { Config } from '@ckb-lumos/config-manager';
+import { SIGN_DATA_MAGIC } from '@nexus-wallet/protocol/lib/constants';
```
Comment (homura) [inline packages/extension-chrome/src/services/ownership/fullOwnership.ts:12]:
```suggestion
import { SIGN_DATA_MAGIC } from '@nexus-wallet/protocol';
```

File: packages/extension-chrome/__tests__/services/ownership/fullOwnership.test.ts (hunk 1 above)
Comment (zhangyouxin) [reply packages/extension-chrome/__tests__/services/ownership/fullOwnership.test.ts]: updated

File: packages/extension-chrome/src/services/ownership/fullOwnership.ts (hunk 2 above)
Comment (zhangyouxin) [reply packages/extension-chrome/src/services/ownership/fullOwnership.ts]: updated

**Comment by homura** [comment]
hi, @zhangyouxin, can you merge with main to avoid workflow failure

**Review by homura — APPROVED** [review]

**Pull request merged**